models/evms/evms_blocks.sql

```diff
@@ -0,0 +1,143 @@
+{{ config(
+        alias ='blocks',
+        unique_key=['blockchain', 'number'],
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum", "polygon", "bnb", "avalanche_c", "gnosis", "fantom", "optimism", "arbitrum"]\',
+                                    "sector",
+                                    "evms",
+                                    \'["hildobby"]\') }}'
+        )
+}}
+
+SELECT 'ethereum' AS blockchain
```

Comment:
any reason we can't use the for loop approach, for cleaner code?

Reply:
if it's due to `blockchain` column, there is a way to do a dictionary variable and assign a value for both blockchain and source table in same variable. rob has some examples out there, likely in nft restructure models

Reply:
same idea across all spells here